src/ralph/back_office/templates/backofficeasset/bo_asset_licence.html

```diff
@@ -0,0 +1,39 @@
+{% extends BASE_TEMPLATE %}
```

Comment:
Yes, we can and we will. But it is not yet ready (Arek working on it).
We wanted to add licenses for demo :)